.ci-mgmt.yaml

```diff
@@ -0,0 +1,16 @@
+template: native
+provider: kubernetes
+lint: true
+aws: true
+gcp: true
```

Comment:
My eyes do a double-take here, can you add a comment about these?

Reply:
Fair enough; these options enable credentials for AWS and GCP respectively so that the provider can run integration tests against those clouds. I've added inline comments to that effect in the file